components/codeflare/codeflare.go

```diff
@@ -83,15 +86,17 @@ func (c *CodeFlare) ReconcileComponent(ctx context.Context, cli client.Client, r
 		}
 
 		if found, err := deploy.OperatorExists(cli, dependentOperator); err != nil {
-			return fmt.Errorf("operator exists throws error %w", err)
+			logger.Error("operator exists throws error", zap.Error(err))
```

Comment:
Handle or return, not both. Wrap the error if it needs local context.

Reply:
ERROR are all set to only print and handle by DSC's logger,  
component level only INFO used by its logger